test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -213,6 +213,43 @@ void AssertIsGam(ITransformer trans)
             Done();
         }
 
+        public class ModelInput
+        {
+#pragma warning disable SA1401
+            public string[] CategoricalFeatures;
+            public float[] NumericalFeatures;
+#pragma warning restore SA1401
+        }
+
+        public class ModelOutput
+        {
+#pragma warning disable SA1401
+            public float[] Score;
+#pragma warning restore SA1401
+        }
+
+
+        [Fact]
+        public void LoadModelWithOptionalColumnTransform()
+        {
+            SchemaDefinition inputSchemaDefinition = SchemaDefinition.Create(typeof(ModelInput));
+            inputSchemaDefinition[nameof(ModelInput.CategoricalFeatures)].ColumnType = new VectorDataViewType(TextDataViewType.Instance, 5);
+            inputSchemaDefinition[nameof(ModelInput.NumericalFeatures)].ColumnType = new VectorDataViewType(NumberDataViewType.Single, 3);
+            var mlContext = new MLContext();
+            ITransformer trainedModel;
+            DataViewSchema dataViewSchema;
+            using (var stream = new FileStream(Path.Combine(Directory.GetCurrentDirectory(), @"..\..\..\..\test\data\backcompat\modelwithoptionalcolumntransform.zip"),
+                FileMode.Open, FileAccess.Read, FileShare.Read))
+            {
+                trainedModel = mlContext.Model.Load(stream, out dataViewSchema);
```

Comment:
>Load [](start = 47, length = 4)

nit: isn't there an overload that takes a file path?

Reply:
This is fine.